nnvm/python/nnvm/frontend/pytorch/aten.py

```diff
@@ -0,0 +1,603 @@
+r'''This file contains one class per PyTorch ATen operator. For the full
+list of operators, see
+https://github.com/zdevito/ATen/blob/master/aten/src/ATen/native/native_functions.yaml
+'''
+import numpy as np
+import tvm
+from nnvm.compiler.graph_util import infer_shape
+from nnvm.symbol import Variable, Symbol
+from nnvm.graph import create
+from nnvm.frontend.common import get_nnvm_op
+from .base import PyTorchOp
+
+
+class ATenOp(PyTorchOp): # pylint: disable=invalid-name
+    r'''Base class for ATen operators'''
+
+    def __init__(self, node, graph):
+        super(ATenOp, self).__init__(node, graph)
+        self.dtype = 'float32'
```

Comment:
Is float32 the only dtype?

Reply:
For simplicity, yes but we can add other datatypes, perhaps in a separate PR. DLR only supports float32.